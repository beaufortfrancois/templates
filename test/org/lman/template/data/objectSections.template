hello
{{#one}}{{foo}} {{bar}}{{/one}}
{{#two}}{{foo}} {{bar}}{{/two}}
goodbye
