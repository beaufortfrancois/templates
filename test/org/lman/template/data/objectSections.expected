hello
a b
c d
goodbye
